src/public/interfaces.ts

```diff
@@ -418,6 +418,11 @@ export interface OpenConversationRequest {
    */
   entityId: string;
 
+  /**
+   * The subtext of the tab
```

Comment:
Can we provide a better description here? I think this should say something like "Description of the item being discussed" or something along those lines.